swan-lake/learn-the-platform/ballerina-tooling/graphql-tool.md

```diff
@@ -244,4 +259,44 @@ This generates a separate Ballerina module (with the project name configured) fo
 2. `utils.bal` - contains all the utility methods related to the client stub
 3. `types.bal`- contains all the Ballerina data types extracted from the GraphQL schema (SDL)
 
-You can use the generated modules to work with multiple GraphQL APIs. Use the client that is generated within each module to execute the queries/mutations on the relevant GraphQL API. Also, use the clients that are generated **within** the same Ballerina package by importing the generated modules. For more information, see [Import a module from the same package](/learn/organize-ballerina-code/#import-a-module-from-the-same-package).
\ No newline at end of file
+You can use the generated modules to work with multiple GraphQL APIs. Use the client that is generated within each module to execute the queries/mutations on the relevant GraphQL API. Also, use the clients that are generated **within** the same Ballerina package by importing the generated modules. For more information, see [Import a module from the same package](/learn/organize-ballerina-code/#import-a-module-from-the-same-package).
+
+## Schema Generation
+
+The GraphQL tool supports generating the GraphQL schema specified by the Schema Definition Language for a given Ballerina GraphQL service. You will be able to export the generated schema to the given location. The tool can be used in the following ways to generate the schema.
```

Comment:
The schema is not specified by the Schema Definition Language. In Ballerina, schema is defined using the Ballerina service. Furthermore, SDL is a generic term, not specific to GraphQL. Shall we reword this accordingly?

```suggestion
The GraphQL tool supports generating the GraphQL schema specified by the Schema Definition Language for a given Ballerina GraphQL service. You will be able to export the generated schema to the given location. The tool can be used in the following ways to generate the schema.
```